build.csx

```diff
@@ -202,7 +201,9 @@ public static void RunDotNetTests(IEnumerable<string> testDirectories)
 
 public static string GetVersion()
 {
-    var version = Regex.Match(File.ReadAllText(assemblyInfo), "AssemblyInformationalVersion\\(\"([^\"]+)\"\\)").Groups[1].Value;
+    var  versionInfo = FileVersionInfo.GetVersionInfo(versionAssembly);
+    var version = versionInfo.FileVersion;
+

```

Comment:
I'll change to `ProductVersion`. On your second note, if 

> `AssemblyInformationalVersion` defaults to `AssemblyFileVersion` if it is not specified. Likewise, `AssemblyInformationalVersion` and `AssemblyFileVersion` default to `AssemblyVersion` if both are not specified.

we could remove two of the attributes!

Of course, I'd rather see us pick up the version from the ReleaseNotes.md file instead and tailor the 3 values accordingly. So maybe we should leave CommonAssemblyInfo.cs alone for now.